code/modules/mob/living/carbon/human/human.dm

```diff
@@ -585,6 +586,16 @@
 	if(href_list["item"])
 		handle_strip(href_list["item"],usr)
 
+	if(href_list["headtail"])
+		usr.visible_message(SPAN_WARNING("\The [usr] is trying to remove something from \the [src]'s headtails!"))
+		if(do_after(usr, HUMAN_STRIP_DELAY, act_target = src))
+			var/obj/item/storage/internal/skrell/S = locate() in src
+			var/obj/item/I = locate() in S
+			if(I)
+				S.remove_from_storage(I, get_turf(src))
+			else
+				to_chat(usr, SPAN_WARNING("\The [src] had nothing in their headtail storage."))
+
```

Comment:
And here you hard-code the button that is being pressed.

This should not be hardcoded, but rather handled on a species level.
I.e. with a additional parameter species_specific in the clicked link which then causes a "species handler" to handle that interaction.